forge-game/src/main/java/forge/game/card/Card.java

```diff
@@ -625,6 +629,29 @@ public boolean changeCardState(final String mode, final String customState, fina
 
             return retResult;
 
+        } else if (mode.equals("Convert") && isConvertable() || hasMergedCard()) {
```

Comment:
You forgot parentheses so this would break all cases below like flipping of mutated cards

Reply:
🤦